.github/workflows/docker-image.yml

```diff
@@ -7,16 +7,17 @@ concurrency:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         include:
-          - image: ubuntu2404
+          - image: fedora39
+          - image: alpine319
```

Comment:
don't we want to build all the images that were changed in this PR?

Reply:
MAINTAINER is a deprecated alias to LABEL        org.opencontainers.image.authors

See: https://docs.docker.com/reference/dockerfile/#maintainer-deprecated